docs/advanced-topics/advschema.md

```diff
@@ -41,11 +41,13 @@ The following storage backends support edge and vertex TTL.
 
 -   HBase
 
+-   BerkeleyDB - supports only hour-discrete TTL those are minimal TTL is one hour.
```

Comment:
Small linguistic nit: I think you mean _supports only hour-discrete TTL, thus the minimal TTL is one hour._ but I'm also not a native speaker.